src/daemon/instance_settings_handler.cpp

```diff
@@ -153,6 +155,27 @@ void update_disk(const QString& key, const QString& val, mp::VirtualMachine& ins
     }
 }
 
+bool is_bridged(const mp::VMSpecs& spec)
+{
+    return std::any_of(spec.extra_interfaces.cbegin(), spec.extra_interfaces.cend(),
+                       [](const auto& network) -> bool { return network.auto_mode; });
```

Comment:
Hey @luis4a0, I think for our purposes we'd better consider the instance bridged if and only if it has an extra_interface corresponding to the network that is currently set for `local.bridged-network`. That would make it symmetric to `launch --bridged` and allow adding multiple interfaces.

Reply:
Hey @ricab, thanks for your comment. You're right, I didn't think on that. I changed the code to reflect this behavior.

Reply:
Great, thanks!